cmake/sysbuild/fast_pair_hex_pm_partition_validation.cmake

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(NOT "${PARTITION_NAME}" STREQUAL "${PARTITION_NAME_DEFINED}")
+  message(FATAL_ERROR
+          "\nThe `bt_fast_pair` partition is not defined in the Partition Manager yaml "
+          "file. Please define the Fast Pair partition to use the automatic provisioning "
```

Comment:
We could emphasize that we need to statically define `bt_fast_pair` PM partition in case Bluetooth Fast Pair is not enable in the application image.

Reply:
addressed